test/run-tests

```diff
@@ -411,19 +416,51 @@ class Task:
                     # we do need to run the setup (if it exists) in the same
                     # invocation of ansible-playbook, so that that it applies
                     # to the same VM as the test playbook.
-                    result = run(
-                        "ansible-playbook",
-                        "-vv",
-                        f"--inventory={inventory}",
-                        setup_file,
-                        playbook,
-                        env={
-                            "TEST_SUBJECTS": image_path,
-                            "TEST_ARTIFACTS": artifactsdir,
-                        },
-                        check=False,
-                        cwd=os.path.dirname(playbook),
-                    )
+                    if backend == CITOOL_BACKEND:
+                        result = run(
+                            "citool",
+                            "-o",
+                            "debug",
```

Comment:
`f"{artifactsdir}/citool-debug.log",`
then we can get per-run citool debug logs